tensorboard/plugins/graph/tf_graph/tf-graph.ts

```diff
@@ -298,12 +302,12 @@ class TfGraph extends LegacyElementMixin(PolymerElement) {
             };
           }),
         });
-        this._setRenderHierarchy(renderGraph);
-        this.async(function () {
-          this.fire('rendered');
-        });
```

Comment:
I found no places where we listen for a 'rendered' event or have `on-rendered` in a template, so I'm assuming this was just unused.

Reply:
I think so. I cannot find any place where we use it.